### PR TITLE
[plugin.video.mlslive] 1.17.3

### PR DESCRIPTION
**Pull request by micahg**
### Description
* Mark as broken due to service shutdown

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by micahg** [comment]
sigh... tyvm.

File: plugin.video.mlslive/addon.xml [hunk 1]
```diff
@@ -17,5 +17,9 @@
     <forum>http://forum.xbmc.org/showthread.php?tid=158142</forum>
     <website>http://wiki.xbmc.org/index.php?title=Add-on:MLS_Live</website>
     <source>https://github.com/micahg/plugin.video.mlslive</source>
+    <news>v1.17.3
+- Marking as broken as MLS Live has shut down</news>
+  </extension>
```
Comment (Rechi) [inline plugin.video.mlslive/addon.xml]: `</extension>` is invalid here

Comment (enen92) [reply plugin.video.mlslive/addon.xml]: lets hope our GSOC student can warn about these kind of issues through schema validation. They are not always easy to find.

Comment (Rechi) [reply plugin.video.mlslive/addon.xml]: look at the travis log, don't know why this was merged with failed CI

Comment (enen92) [reply plugin.video.mlslive/addon.xml]: Fixed by lunatixz in https://github.com/xbmc/repo-plugins/pull/1875#merged-event

**Pull request merged**